### PR TITLE
Created my LinkFree profile

### PR DESCRIPTION
**Pull request by Shield01**
## Changes proposed

My PR creates my LinkFree profile.

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Screenshots

image

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2571"></a>

### CODE REVIEW

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Hey Shield01, adding profile doesn't require changes in lock files. Please revert the changes in `package-lock.json` file. Let us know if you need any help with this! 🛠

**Comment by Shield01** [comment]
> Hey Shield01, adding profile doesn't require changes in lock files. Please revert the changes in `package-lock.json` file. Let us know if you need any help with this! 🛠

Thanks kumarsonsoff3, this feedback is appreciated. I will make the requested changes and make a new pull request.

**Comment by eddiejaoude** [comment]
You can make a new PR if easier (please close this one if you create another PR), or you can add more changes to this PR and undo your changes to the package lock file with this command 

```
git checkout main -- package-lock.json
```

then commit and push the changes and your PR should no longer show any changes to the lock file

I hope that helps

**Comment by Shield01** [comment]
> You can make a new PR if easier (please close this one if you create another PR), or you can add more changes to this PR and undo your changes to the package lock file with this command
> 
> ```
> git checkout main -- package-lock.json
> ```
> 
> then commit and push the changes and your PR should no longer show any changes to the lock file
> 
> I hope that helps

Thank you very much eddiejaoude, I'll do this right away.

**Comment by Shield01** [comment]
> You can make a new PR if easier (please close this one if you create another PR), or you can add more changes to this PR and undo your changes to the package lock file with this command
> 
> ```
> git checkout main -- package-lock.json
> ```
> 
> then commit and push the changes and your PR should no longer show any changes to the lock file
> 
> I hope that helps

Thank you very much eddiejaoude, I'll do this right away.

**Comment by Shield01** [comment]
Thanks, eddiejaoude. I followed your advice.
Your guidance is highly appreciated.

File: data/Shield01.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    },
+    {
+      "name": "CV: Read about me",
+      "url": "https://docs.google.com/document/d/1HlBIWpg2UCgHuUl12mCVRDWB6vmeGCwmp6Oxx1s0a58/edit?usp=sharing",
+      "icon": "FaLink"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Open Source project Linkfree",
```
Comment (Panquesito7) [inline data/Shield01.json:48]: You might want to update this field.

Comment (eddiejaoude) [reply data/Shield01.json:48]: Maybe something like `Create my Profile on the Open Source project LinkFree`?

Comment (Shield01) [reply data/Shield01.json:48]: Oops, yeah, I agree... "started" implies that I am the founder.
Sincere apologies for this.

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍

**Comment by Shield01** [comment]
> Looks good 👍

Thanks eddiejaoude

**Pull request merged**